Formula/qsoas.rb

```diff
@@ -50,11 +56,17 @@ def install
                     "QMAKE_LFLAGS=-L#{libexec}/lib -L#{gsl}/lib"
     system "make"
 
-    prefix.install "QSoas.app"
-    bin.write_exec_script "#{prefix}/QSoas.app/Contents/MacOS/QSoas"
+    if OS.mac?
+      prefix.install "QSoas.app"
+      bin.write_exec_script "#{prefix}/QSoas.app/Contents/MacOS/QSoas"
+    else
+      bin.install "QSoas"
+    end
   end
 
   test do
+    # Set QT_QPA_PLATFORM to minimal to avoid error "qt.qpa.xcb: could not connect to display"
+    ENV["QT_QPA_PLATFORM"] = "minimal" unless OS.mac?
```

Comment:
Does this not work on macOS?

Reply:
The error doesn't happen on macOS because we do have a working display on that platform.  I figured it would be better to use the full graphics stack when it's available.  Eventually I hope we'll have some sort of fake X display setup in Linux as well which would let us get rid of these workarounds.